pkg/ansible/paramconv/paramconv.go

```diff
@@ -70,6 +71,42 @@ func ToCamel(s string) string {
 	return ret
 }
 
+// This function modifies the string to handle special words that are a part of wordMapping
+// Add values to handle wider number of cases
+func preprocessWordMapping(s string) string {
+	count := 0
+	length := len(s)
+	for i, v := range s {
+		if i+1 >= length {
+			break
+		}
+		next := s[i+1]
+		count++
+		if (v >= 'A' && v <= 'Z' && next >= 'a' && next <= 'z') || (v >= 'a' && v <= 'z' && next >= 'A' && next <= 'Z') {
```

Comment:
for readability's sake, maybe add some util functions for checking if a character is upper or lower cased?